src/index.js

```diff
@@ -75,128 +75,191 @@ export default class OneSphere {
   }
 
   getSession(args) {
-    const params = args ? getUrlParams(args) : '';
+    if (args && (typeof args !== 'object' || Array.isArray(args))) {
+      throw new Error('args parameter should be an object');
+    }
+    const params = getUrlParams(args);
```

Comment:
Each instance of this block should be adjusted to something like:
`const params = handleUrlParams(args);`

`handleUrlParams` would look something like 

```javascript
const handleUrlParams = args => {
  if (!args || typeof args !== 'object' || Array.isArray(args)) {
    return throw new Error('args parameter should be an object');
  }

  return getUrlParams(args);
});
```

If we ever need to change how we handle params we only need to change it in one place.

Reply:
Indeed, sorry for that I went too fast. Updating now 😅